transport/http/outbound.go

```diff
@@ -399,6 +405,11 @@ func getYARPCErrorFromResponse(response *http.Response, bothResponseError bool)
 	)
 }
 
+func checkServiceMatch(reqSvcName string, resHeaders *http.Header) (bool, string) {
```

Comment:
comment about this only does verification if there's a response header

Reply:
Added.